contrib/sssd-tmpfiles.conf.in

```diff
@@ -0,0 +1 @@
+d /run/sssd 775 sssd sssd -
```

Comment:
Hi,

I wonder if you wanted to use `d /run/sssd 775 @SSSD_USER@ @SSSD_USER@ -`? Otherwise I think you do not have to add it to ` AC_CONFIG_FILES`.

bye,
Sumit

Reply:
Make sense indeed. Thanks, updated.

Reply:
Thanks